cpp/src/jni/dataset/jni_util.cc

```diff
@@ -16,36 +16,44 @@
 // under the License.
 
 #include "jni/dataset/jni_util.h"
-
+#include "arrow/ipc/metadata_internal.h"
+#include "arrow/util/base64.h"
+#include "arrow/util/key_value_metadata.h"
 #include "arrow/util/logging.h"
 
+#include <memory>
 #include <mutex>
 
+#include <flatbuffers/flatbuffers.h>
+
 namespace arrow {
+
+namespace flatbuf = org::apache::arrow::flatbuf;
```

Comment:
nit: can you prefix org with "::"

Reply:
Thanks but did you mean to use `namespace flatbuf = ::org::apache::arrow::flatbuf;`? I did a search in our C++ code base using RegEx `namespace \w* = `, and it seems to be a common practice not adding the leading `::`
```sh
[root@localhost arrow]# grep -R "namespace [a-z A-Z][a-z A-Z]* =.*" cpp/src/
cpp/src/parquet/column_reader.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/parquet/column_writer.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/parquet/encoding_test.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/parquet/arrow/reader.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/parquet/arrow/reader_internal.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/parquet/encoding.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/parquet/column_writer_test.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/parquet/statistics_test.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/jni/dataset/jni_util.cc:namespace flatbuf = org::apache::arrow::flatbuf;
cpp/src/arrow/ipc/json_simple.cc:namespace rj = arrow::rapidjson;
cpp/src/arrow/ipc/metadata_internal.cc:namespace flatbuf = org::apache::arrow::flatbuf;
cpp/src/arrow/ipc/reader.cc:namespace flatbuf = org::apache::arrow::flatbuf;
cpp/src/arrow/ipc/metadata_internal.h:namespace flatbuf = org::apache::arrow::flatbuf;
cpp/src/arrow/compute/kernels/aggregate_benchmark.cc:namespace BitUtil = arrow::BitUtil;
cpp/src/arrow/filesystem/s3_test_util.h:namespace bp = boost::process;
cpp/src/arrow/filesystem/s3fs_test.cc:namespace bp = boost::process;
cpp/src/arrow/gpu/cuda_arrow_ipc.cc:namespace flatbuf = org::apache::arrow::flatbuf;
cpp/src/arrow/flight/flight_benchmark.cc:namespace perf = arrow::flight::perf;
cpp/src/arrow/flight/test_util.cc:namespace bp = boost::process;
cpp/src/arrow/flight/test_util.cc:namespace fs = boost::filesystem;
cpp/src/arrow/flight/test_util.cc:  namespace fs = boost::filesystem;
cpp/src/arrow/flight/internal.h:namespace pb = arrow::flight::protocol;
cpp/src/arrow/flight/server.cc:namespace pb = arrow::flight::protocol;
cpp/src/arrow/flight/serialization_internal.cc:namespace pb = arrow::flight::protocol;
cpp/src/arrow/flight/client.cc:namespace pb = arrow::flight::protocol;
cpp/src/arrow/flight/client.cc:          namespace ge = GRPC_NAMESPACE_FOR_TLS_CREDENTIALS_OPTIONS;
cpp/src/arrow/flight/perf_server.cc:namespace perf = arrow::flight::perf;
cpp/src/arrow/flight/perf_server.cc:namespace proto = arrow::flight::protocol;
cpp/src/arrow/flight/flight_test.cc:namespace pb = arrow::flight::protocol;
cpp/src/arrow/adapters/orc/adapter.cc:namespace liborc = orc;
cpp/src/arrow/adapters/orc/adapter_util.h:namespace liborc = orc;
cpp/src/arrow/adapters/orc/adapter_test.cc:namespace liborc = orc;
cpp/src/arrow/adapters/orc/adapter_util.cc:namespace liborc = orc;
cpp/src/arrow/json/object_parser.cc:namespace rj = arrow::rapidjson;
cpp/src/arrow/json/test_common.h:namespace rj = arrow::rapidjson;
cpp/src/arrow/json/chunker.cc:namespace rj = arrow::rapidjson;
cpp/src/arrow/json/parser.cc:namespace rj = arrow::rapidjson;
cpp/src/arrow/json/object_writer.cc:namespace rj = arrow::rapidjson;
cpp/src/arrow/testing/json_internal.h:namespace rj = arrow::rapidjson;
cpp/src/plasma/store.cc:namespace fb = plasma::flatbuf;
cpp/src/plasma/client.cc:namespace fb = plasma::flatbuf;
cpp/src/plasma/common.cc:namespace fb = plasma::flatbuf;
cpp/src/plasma/plasma.cc:namespace fb = plasma::flatbuf;
cpp/src/plasma/test/serialization_tests.cc:namespace fb = plasma::flatbuf;
cpp/src/plasma/protocol.cc:namespace fb = plasma::flatbuf;
```